src/access/wac.test.ts

```diff
@@ -515,3 +515,372 @@ describe("getAgentAccess", () => {
     });
   });
 });
+
+describe("getGroupAccess", () => {
+  it("calls the included fetcher by default", async () => {
+    const mockedFetcher = jest.requireMock("../fetcher.ts") as {
+      fetch: jest.Mock<
+        ReturnType<typeof window.fetch>,
+        [RequestInfo, RequestInit?]
+      >;
+    };
+
+    const resource = getMockDataset(
+      "https://some.pod/resource",
+      "https://some.pod/resource.acl"
+    );
+    await getGroupAccess(resource, "https://some.pod/groups#group");
+
+    expect(mockedFetcher.fetch.mock.calls[0][0]).toEqual(
+      "https://some.pod/resource.acl"
+    );
+  });
+
+  it("returns null if no ACL is accessible", async () => {
+    const mockFetch = jest
+      .fn(window.fetch)
+      // No resource ACL available...
+      .mockResolvedValueOnce(
+        mockResponse("", {
+          status: 404,
+          url: "https://some.pod/resource.acl",
+        })
+        // Link to the fallback ACL...
```

Comment:
These comments are weirdly aligned with the code they're about again...

Reply:
Damn, I'm not sure if it's prettier or a merge issue. i'll make sure to fix it in https://github.com/inrupt/solid-client-js/pull/787